app/main.py

```diff
@@ -16,4 +16,9 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+
+sorted_variables = {"mutable": [my_favourite_films, marks,
+                                collection_of_coins],
+                    "immutable": [lucky_number, pi, one_is_a_prime_number,
+                                  name, profile_info]
+                    }
```

Comment:
please, use a code style as mentioned in this file:
https://github.com/mate-academy/py-linter-formatter/blob/master/checklist.md

The braces must be located in one of two options: open and start with the text or have a line break between the text